cmd/pj-rehearse/main.go

```diff
@@ -281,21 +283,22 @@ func rehearseMain() int {
 	metrics.RecordOpportunity(toRehearseClusterProfiles, "cluster-profile-change")
 	toRehearse.AddAll(toRehearseClusterProfiles)
 
-	rehearsals := rehearse.ConfigureRehearsalJobs(toRehearse, prConfig.CiOperator, prNumber, loggers, o.allowVolumes, changedTemplates, changedClusterProfiles)
-	metrics.RecordActual(rehearsals)
-	if len(rehearsals) == 0 {
+	jobConfigurer := rehearse.NewJobConfigurer(toRehearse, changedPeriodics, prConfig.CiOperator, prNumber, loggers, o.allowVolumes, changedTemplates, changedClusterProfiles)
+	presubmits, periodics := jobConfigurer.ConfigureRehearsalJobs()
+	metrics.RecordActual(presubmits)
+	if len(presubmits) == 0 && len(changedPeriodics) == 0 {
 		logger.Info("no jobs to rehearse have been found")
 		return 0
-	} else if len(rehearsals) > o.rehearsalLimit {
+	} else if len(presubmits) > o.rehearsalLimit {
```

Comment:
Does not seem right, I think we should count both job types against the threshold, so we never rehearse more than the $LIMIT jobs.